File: src/main/java/com/sequenceiq/cloudbreak/repository/CloudbreakUsageRepository.java
package com.sequenceiq.cloudbreak.repository;

import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
import org.springframework.data.repository.CrudRepository;

import com.sequenceiq.cloudbreak.domain.CloudbreakUsage;

public interface CloudbreakUsageRepository extends CrudRepository<CloudbreakUsage, Long>, JpaSpecificationExecutor {
}

File: core/src/main/java/com/sequenceiq/cloudbreak/repository/CloudbreakUsageRepository.kt
package com.sequenceiq.cloudbreak.repository

import org.springframework.data.jpa.repository.JpaSpecificationExecutor
import org.springframework.data.repository.CrudRepository

import com.sequenceiq.cloudbreak.domain.CloudbreakUsage

@EntityType(entityClass = CloudbreakUsage::class)
interface CloudbreakUsageRepository : CrudRepository<CloudbreakUsage, Long>, JpaSpecificationExecutor<Any>
